### PR TITLE
Add a note about using Composer to install Encore

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: frontend/encore/installation.rst [hunk 1]
```diff
@@ -25,6 +25,14 @@ is also created/updated. When using npm 5, a ``package-lock.json`` file is creat
 
 Next, create your ``webpack.config.js`` in :doc:`/frontend/encore/simple-example`!
 
+.. tip::
+
+    If you are using Flex for your project, you can install Encore via:
+
+.. code-block:: terminal
+
+    $ composer require encore
```
Comment (xabbuh) [inline frontend/encore/installation.rst]: The code block needs to be indented by four spaces if it should be part of the tip.

Comment (fabpot) [reply frontend/encore/installation.rst]: indeed, fixed

**Review by xabbuh — APPROVED** [review]
Just another minor comment: Maybe we should move this tip right below the note above that mentions how to install Encore using npm.

**Comment by fabpot** [comment]
Make sense, moved the note.

**Comment by xabbuh** [comment]
Thank you @fabpot.

**Pull request merged**